### PR TITLE
chore: Take version from latest git tag, correct version in package.json

### PR DESCRIPTION
**Pull request by mayurnewase**
Update version in package.json

### TESTING INSTRUCTIONS

Install superset
Start server
Check version shown on welcome page under setting dropdown.

### ADDITIONAL INFORMATION

- [x] Has associated issue: https://github.com/apache/superset/issues/13674
- [ ] Required feature flags:
- [ ] Changes UI
- [ ] Includes DB Migration (follow approval process in [SIP-59](https://github.com/apache/superset/issues/13351))
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -25,13 +25,22 @@
...
 PACKAGE_JSON = os.path.join(BASE_DIR, "superset-frontend", "package.json")
-with open(PACKAGE_JSON, "r") as package_file:
-    version_string = json.load(package_file)["version"]
 
 with io.open("README.md", "r", encoding="utf-8") as f:
     long_description = f.read()
 
 
+def get_version() -> str:
+    version_string = "1.3.2"  # current latest as last fallback
```
Comment (mayurnewase) [inline setup.py:34]: this may not be used.

**Comment by junlincc** [comment]
Thank you so much @mayurnewase ! will test it locally.

**Comment by junlincc** [comment]
looks like it only show the helm chart version? is it possible to show either 1.3.2 or latest master? 
Screen Shot 2021-11-12 at 2 52 35 PM

**Comment by mayurnewase** [comment]
it will take from latest release tag from github.
will update it to take it only what we put in package.json file,so we can put whatever we want.

**Comment by junlincc** [comment]
@mayurnewase thank you! lmk when it's ready 🙏

**Comment by mayurnewase** [comment]
@junlincc now it should show 1.3.2

**Review by villebro — CHANGES_REQUESTED** [review]
It seems the git-based automation was removed. I think it's important to distinguish a real release (1.3.2) from master branch or a custom tag/branch. I kinda liked using git for it; was there a reason for reverting that code?

**Comment by mayurnewase** [comment]
@villebro  I had used `git describe --tags --abbrev=0` to get latest git tag available on current branch, but it showing helm charts tags as they were latest.
Should we do similar and filter by regex to show latest superset release tag?

**Comment by mayurnewase** [comment]
Something like `git tag --sort=committerdate | tail -50 | grep -e "^\d" | tail -1`

**Pull request closed without merging**